crates/protect-ffi/src/lib.rs

```diff
@@ -388,6 +405,8 @@ async fn decrypt_bulk(
         .zerokms
         .decrypt(
             encrypted_records,
+            // Specifying None here will result in the client using the keyset identifier from the client
```

Comment:
Or the keyset ID encoded into the ciphertext if present.

Reply:
The type returned by calls to encrypt is an `EncryptedRecord` which includes the `keyset_id` if one was used. See https://docs.rs/cipherstash-client/0.28.0/cipherstash_client/zerokms/struct.EncryptedRecord.html

Reply:
Re above, I think this can be addressed as a follow up feature

Reply:
I think you've already addressed it.